### ISSUE TITLE
Loss Functions

### ISSUE THREAD

**Issue by SprD12** [zijianchen98/AGIN#1]
Hello, 
In the utils folder, function.py file, different loss functions code are given (Fidelity Loss, Monotonicity loss). I just wanted to know which equation of the paper does each of these loss correspond to?

**Comment by Codeblade98** [comment]
Also it would be great if the authors (@zijianchen98 ) provide the implementation of the WSD Loss function which was used to train the model, since the current codebase just has an implementation based on VGG while the feature extractor of the rationality branch is ResNet based
